tests/Helper/OutputResultConverterTest.php

```diff
@@ -327,17 +310,17 @@ public function testFull(): void
                 ],
                 'artifacts' => [
                     'uploaded' => [
-                        ['storageFileId' => 12344],
-                        ['storageFileId' => 12347],
-                        ['storageFileId' => 23456],
-                        ['storageFileId' => 23480],
+                        ['storageFileId' => '12344'],
+                        ['storageFileId' => '12347'],
+                        ['storageFileId' => '23456'],
+                        ['storageFileId' => '23480'],
                     ],
                     'downloaded' => [
-                        ['storageFileId' => 12345],
-                        ['storageFileId' => 12346],
-                        ['storageFileId' => 23467],
-                        ['storageFileId' => 23478],
-                        ['storageFileId' => 23479],
```

Comment:
Tady jsem to zmenil na stringy, protoze me zmatli testy v Internal Clientovi, ale kdyz to tady bylo jako int, tak to tak muzu nechat a z clienta to pretypovani vyhodit, co myslis @odinuv?

https://github.com/keboola/job-queue/pull/237/files#diff-aa238e5e09e68cf468d9db1f4eb923cc203e3100db473f2447d7c95e4f72c716R55

Reply:
Dobre, diky